examples/bits.rs

```diff
@@ -90,22 +107,18 @@ fn main() {
 
     // `bitvec` can look at more than just `u8`. Let's try looking at the bits
     // that represent a number!
-
-    // First we need to pick a number. Rust provides some interesting numbers
-    // for us.
-    let num = std::f64::consts::PI * 2.0;
-
+    //
     // But `bitvec` doesn't know how to view all numbers. The standard library
     // provides a function `to_bits(f64) -> u64`, which turns it into a number
     // `bitvec` does know how to view.
     //
     // The `f64` data type has a memory encoding standardized by the IEEE-754
     // document. You can read more about it here:
     // https://en.wikipedia.org/wiki/Double-precision_floating-point_format
-    let raw_number = num.to_bits();
+    let raw_number: u64 = NUMBER.to_bits();
```

Comment:
The standard library's function name makes sense when looking at only the standard library, but conflicts visually with `Bits::bits` in `bitvec`, so I put `: u64` here in order to clarify that this is only changing the type, not producing a `bitvec` view.

Reply:
Sounds good!